pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -101,6 +101,46 @@ type CiliumBGPNeighborGracefulRestart struct {
 	RestartTimeSeconds *int32 `json:"restartTimeSeconds,omitempty"`
 }
 
+// BGPCommunityString holds a string containing a BGP Community Attribute value (RFC 1997)
+// as a 4-byte decimal number or two 2-byte decimal numbers separated by a colon,
+// or a BGP Large Community Attribute value (RFC 8092) as three 4-byte decimal numbers separated by a colon.
+// +kubebuilder:validation:Pattern=`^\d+(:\d+)?(:\d+)?$`
+type BGPCommunityString string
+
+// CiliumBGPRouteAttributes can be used to apply additional path attributes
+// to matched routes when advertising them to a BGP peer.
+type CiliumBGPRouteAttributes struct {
+	// SelectorType defines type of the object on which the Selector applies.
+	// - For "Node" the Selector matches k8s node resources (resulting into PodCIDR routes).
```

Comment:
> Will this always just select PodCIDR routers? 

I was a bit forward-thinking about [Multi-Pool IPAM](https://docs.cilium.io/en/v1.14/network/kubernetes/ipam-multi-pool/), but for that it may be a better idea to select `CiliumPodIPPool` resources instead. So yeah, could be just PodCIDR after all.

> Or is the reason this is 'Node' to better align with the 'Selector' as you'd put the label on the node?

Right, as this is called SelectorType, I think it should suggest which type the `Selector` is actually selecting (which BTW is incorrect ATM in the description, it should be `CiliumNode` - I have already renamed this one myself few times :joy: ).

I'm not super-happy abut this name myself TBH. Maybe `PodCIDRNode`? But I am also fine with just `PodCIDR`.

Reply:
Tough one, I can see the usefulness of both when mixed with NodeSelector, but in the sake of "calling things what they are" PodCIDR may make the most sense. Your call tho.

Reply:
"calling things what they are" makes sense, will change it to `PodCIDR`

Reply:
renamed to `PodCIDR`